_posts/2018-10-01-why-developers-do-not-like-change.md

```diff
@@ -0,0 +1,75 @@
+house every week, you would probably not be overly enamoured by the idea. We seek stability the majority of the time.
+
+This is not to say we do not seek change from time to time:
+
+ * Move house
+ * Change job
+ * Re-decorate a room
+ * Change your car
+
+## No one likes to lose
+From birth, it is pre-determined that we do not like losing. In the brilliant book 
+[Thinking, Fast and Slow](https://www.amazon.co.uk/Thinking-Fast-Slow-Daniel-Kahneman/dp/0141033576) talks about gains
+and losses. I won't delve into the details of Prospect Theory here, but if you want to understand more I suggest you
+read the book.
+
+In the book Kahneman explains how we react worse to losses than to gains, and we are much more sensitive to losses.
+This fear of losing can help explain why developers are not fond of organisational changes.
+
+For example, take the below scenario:
+ 
+
+> Company X has an software engineering department that consists of ten developers.
+
+
+Company X wish to make some changes, if you were one of those developers, which on of the below would you prefer?
+
+> A - Company X proposes a change in which there is a 70% chance you will lose your job
+
+> B - Company X proposes a change in which there is a 5% chance you will lose your job
+
+
+If you are like the majority of people, you will prefer option `B`, despite the fact that both options present potential
+losses (in this case, your job).
+
+Unfortunately, when changes are proposed the facts are not known and so you would not be able to make such a decision as
+above. There normally is a large amount of uncertainty, and your decision to remain at a company will be a gamble.
+
+## The illusion of moving
+When presented with an organisational change marked with much uncertainty, many think that moving company will be the
+solution. However, this is an illusion and can often result in people hastily jumping ship and then having to do the
+same at their new company because the grass was not in fact greener.
+
+## Creatures of habit
+As developers, we are creatures of habit. Much of the technologies we like is formed due to our familiarity with such
+technologies. For example, say your favourite programming language is Java, this will most likely be because you have
+a lot of experience using it and you have good memories programming in Java.
+
+Now say your bosses tell you that you must now use C# instead, this conflicts with your passion for Java. You most
+likely view this change as a loss, not a gain. Therefore you will most likely make a fuss and voice your dissatisfaction
+of using C#.
+
+## My advice to developers
+In the event of you finding yourself in the middle of a proposed organisational change, seek out as much information as
+possible to help you make your decision. See what you have to gain or lose from the change. If there is not much to lose
+then I would recommend seeing how the change goes, you might find yourself in a situation where you can learn a lot.
```

Comment:
Could expand on this with the following points: 

 * When developers don't like a change they should communicate this to management, not complain about it amongst themselves. 
 * Feedback should be constructive & specific (points 3-5 here might be relevant: https://personalexcellence.co/blog/constructive-criticism/)

Reply:
Yes, I think I will concentrate on this. My initial post was just to get a basic structure up and get some early feedback.

I will definitely include some of the points you raise as I have experienced similar.